sites/public/styles/sign-in.module.scss

```diff
@@ -0,0 +1,6 @@
+.forgot-password {
+    @apply float-right;
```

Comment:
We're moving away from using tailwind, so these should be added as vanilla CSS or using seeds variables.